build.gradle

```diff
@@ -16,7 +16,8 @@ repositories {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter:3.0.0'
-	testImplementation 'org.springframework.boot:spring-boot-starter-test:3.0.0'
+    implementation 'org.jetbrains:annotations:20.1.0'
```

Comment:
You can you @Nonnull from Lombok.

Reply:
do I nead to delete jetbrains  dependencies

Reply:
yes I have checked where I use it and deleted it